internal/cri/server/container_status.go

```diff
@@ -91,9 +91,10 @@ func toCRIContainerStatus(ctx context.Context, container containerstore.Containe
 	status := container.Status.Get()
 	reason := status.Reason
 	if status.State() == runtime.ContainerState_CONTAINER_EXITED && reason == "" {
-		if status.ExitCode == 0 {
+		switch status.ExitCode { //nolint:QF1003
+		case 0:
 			reason = completeExitReason
-		} else {
+		default:
```

Comment:
This seems like a superfluous change at first glance. Why was this changed, i.e., what are you trying to enhance/fix?